app/views/group_offers/index.html.slim

```diff
@@ -1,6 +1,12 @@
 h1= t_title
 
-= form_navigation_btn :new
+nav.navbar.section-navigation
+  ul.list-inline
+    li= button_link t_title(:new), new_group_offer_path, 'success', dimension: 'sm'
```

Comment:
Since this button is now `sm`, I think it would look better if the new offer button at the end of the page was also small.